Drop-In/src/main/java/com/braintreepayments/api/SupportedPaymentMethodsFragment.java

```diff
@@ -111,19 +106,22 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             }
         });
 
+        dropInViewModel.getUserCanceledError().observe(getViewLifecycleOwner(), exception -> {
```

Comment:
We discovered a bug in the 3DS vaulted payment cancel flow - when a user selects a vaulted 3DS card, then cancels from the 3DS flow, the loading indicator is never removed. This update allows the `SupportedPaymentMethodsFragment` to listen for those cancellations and update the UI.